DEVELOPMENT.md

```diff
@@ -33,35 +26,34 @@ npm install
 
 #### Creating your `.env` file
 
-create a `.env` file like this:
+Create a `.env` file like this (making the substitution described in `<…>`):
 
 ```
 PORT=3233
 DATABASE_URL=postgres://localhost:5432/lg-curriculum
 IDM_BASE_URL=http://idm.learnersguild.dev
 ECHO_BASE_URL=http://echo.learnersguild.dev
-JWT_PUBLIC_KEY="SEE IDM SETUP"
+JWT_PUBLIC_KEY="<get from .env.development file in your IDM directory>"
 LOG_SQL_QUERIES=1
+# Uncomment the next line if you want to disable authentication to IDM.
+# DISABLE_IDM=1
 ```
 
-_NOTE: you can set `DISABLE_IDM=1` to disable authentication to IDM_
-
-
-#### Setup the Postgresql Database
+#### Set up the Postgresql Database
 
 ```sh
 createdb lg-curriculum
 npm run db:migrate
 ```
 
-
 #### Start the server in development mode
 
 ```sh
 npm run dev
 ```
 
+#### Visit the server in your browser and (if necessary) sign in with Github:
 
-## Continuous Integration
+http://curriculum.learnersguild.dev
 
-https://circleci.com/gh/GuildCrafts/curriculum
```

Comment:
I'd like to keep this. This is actually a good link if you have the permissions.